app/assets/javascripts/turbolinks.js

```diff
@@ -0,0 +1,11 @@
+$(function() {
```

Comment:
Looks like something here is causing an error on the gametime switch toggle causing it to not activate.  The faq page also throws a js error but nothing appears broken on it.

Reply:
Looks like the FAQ page thing also happens on the current live site so it unrelated to this change. I'll make an issue for it separately.

Looking into the game switch thing.

Reply:
Oh, looks like its a naming conflict.  Change the name of this file and the error goes away.  I assume the include in the application.js file is loading this as it is checked first in the pipeline.

Reply:
Yep, I figured it out at the same time you did lol. It tries to load Turbolinks the library and ends up loading my turbolinks.js instead. Oh Rails.